### DISCUSSION TITLE
Is Optimise intended to be used in prod or only dev?

### DISCUSSION THREAD

**Discussion by elie222** [prisma/optimize-feedback#4] [Q&A]
The docs seemed to suggest only dev, but the example code enables it for both.

**Comment by Varkoff** [comment]
I'm having a hard time enabling it in production. In dev, I don't have a lot of data, so I think it's irrelevant to enable it there.

I'm stuck at the CLI command to authenticate the project on the VPS, as it opens a random port to authorize the request. Using Caddy Server, I do not know how to 'open all ports' and forward them to localhost.

Asked for help on Discord https://discord.com/channels/937751382725886062/1240949417008304188/1240949417008304188

**Comment by millsp** [comment]
At the moment, Prisma Optimize is a tool dedicated for local development. Nothing prevents you using it in prod, but this use case is not something we officially cater for and you may or may not encounter issues. The CLI commands are not required anymore. We have reworked this and we now have a concept of API tokens. Read along over here https://www.prisma.io/data-platform/optimize.
